test/__snapshots__/index.test.js.snap

```diff
@@ -1481,7 +1481,7 @@ exports[`Validation should fail validation for oneOf #1 1`] = `
 exports[`Validation should fail validation for oneOf #2 1`] = `
 "Invalid configuration object. Object has been initialised using a configuration object that does not match the API schema.
  - configuration.optimization.runtimeChunk should be one of these:
-   boolean | \\"single\\" | \\"multiple\\" | object { name? }
+   boolean | \\"single\\" | \\"multiple\\" | empty string | object { name? }
```

Comment:
`empty string`, regression?

Reply:
what do you mean? it is by design, when `maxLength: 0`

Reply:
Oh, I understood what you mean, I will check

Reply:
@evilebottnawi I have updated `anyOf` in `configuration.optimization.runtimeChunk` with
```json
{
              "type": "string",
              "maxLength": 0
            },
```

Reply:
:+1: